src/components/Sidebar/common/SectionResult.test.js

```diff
@@ -47,12 +47,15 @@ describe('SectionResult', () => {
 
   it('turns scheduled text to red if section is unscheduled', () => {
     timeUtils.getFormattedClassSchedule.mockReturnValue('TBA');
+    timeUtils.isUnscheduled.mockReturnValue(true);
```

Comment:
For all the other places that use this imported function, the return value should also be mocked (good unit testing technique).

Reply:
Done